protocol/contracts/libraries/Minting/LibWellMinting.sol

```diff
@@ -183,6 +184,56 @@ library LibWellMinting {
         }
     }
 
+    function cappedReservesDeltaB(address well) internal view returns (int256) {
+        if (well == C.BEAN) {
+            return 0;
+        }
```

Comment:
can we generalize this to use `LibWell.isWell()` instead?

Reply:
Yes, fixed here: https://github.com/BeanstalkFarms/Beanstalk/commit/f06a7d55531227a38c7d29d54e4687aa76a8d4ee